jcommune-view/jcommune-web-view/src/main/webapp/resources/javascript/lib/wysiwyg-bbcode/editor.js

```diff
@@ -82,6 +82,7 @@ function initEditor(textAreaId, htmlAreaId, baseDivId) {
  * preview mode.
  */
 function togglePreviewMode(allowedUrls) {
+    addPollItemButton = $("input#addPollButton");
```

Comment:
#pollButton ?  you change name or it is another button. and check in another places